### ISSUE TITLE
No text label displayed on Google Colab (Pytorch 1.8.1)

### ISSUE THREAD

**Issue by makaishi2** [szagoruyko/pytorchviz#58]
I run following sample code on both local anaconda and google colab.
And on google colab no text label dispkayed on a result node.

Sample Code:

```import numpy as np
import torch
from torchviz import make_dot
print(torch.__version__)
x_np = np.arange(-2, 2.1, 0.25)
x = torch.tensor(x_np, requires_grad=True)
y = 2 * x**2 + 2
z = y.sum()
params = dict(x = x)
make_dot(z, params=params)
```

Result 1 (local anaconda pytorch 1.7.1)
<img width="166" alt="スクリーンショット 2021-04-03 13 08 54" src="https://user-images.githubusercontent.com/25073940/113467448-0116e580-947e-11eb-8721-8dcfbad6d979.png">

Result 2 (Google Colab pytorch 1.8.1)
<img width="153" alt="スクリーンショット 2021-04-03 13 09 15" src="https://user-images.githubusercontent.com/25073940/113467454-08d68a00-947e-11eb-8028-5508f5f06f13.png">

**Comment by albanD** [comment]
Hi,

As mentioned in the doc here: https://github.com/szagoruyko/pytorchviz/blob/5c91c3d52451f22af1a388993b798eaac1e0a74c/torchviz/dot.py#L50

The green node is the Tensor passed as output. The `()` is its size.
You can pass `z=z` to the params dict and that name will be displayed there.
